manifests/k/KhronosGroup/VulkanSDK/1.3.231.1/KhronosGroup.VulkanSDK.installer.yaml

```diff
@@ -0,0 +1,23 @@
+# Created with YamlCreate.ps1 v2.2.1 $debug=NVS2.CRLF.7-3-0.Win32NT
+# yaml-language-server: $schema=https://aka.ms/winget-manifest.installer.1.2.0.schema.json
+
+PackageIdentifier: KhronosGroup.VulkanSDK
+PackageVersion: 1.3.231.1
+InstallerLocale: en-US
+MinimumOSVersion: 10.0.0.0
+InstallerType: exe
+Scope: machine
+InstallModes:
+- interactive
+- silent
+InstallerSwitches:
+  Silent: --accept-licenses --default-answer --confirm-command install
+  SilentWithProgress: --accept-licenses --default-answer --confirm-command install
+UpgradeBehavior: install
+ReleaseDate: 2022-10-26
+Installers:
```

Comment:
```suggestion
Dependencies:
  PackageDependencies:
    - PackageIdentifier: Microsoft.VCRedist.2015+.x64
    - PackageIdentifier: Microsoft.VCRedist.2015+.x86
Installers:
```

Reply:
Sad the [Lunar docs](https://vulkan.lunarg.com/doc/sdk/1.3.231.1/windows/getting_started.html) don't mention dependency on the 2015+ Visual C Redistribution.